samples/matter/light_bulb/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# TODO: Workaround to be removed once PWM will be supported on nRF54L.
+CONFIG_PWM=n
+
+# Multirole is the only currently supported role by SoftDevice.
+CONFIG_BT_LL_SOFTDEVICE_MULTIROLE=y
+
+# TODO: Workaround to be removed once DFU will be supported on nRF54L.
+CONFIG_CHIP_OTA_REQUESTOR=n
+CONFIG_CHIP_QSPI_NOR=n
+
+CONFIG_FPROTECT=n
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_FPU=n
+CONFIG_PM=n
+CONFIG_HWINFO_NRF=n
+
+CONFIG_NRF_802154_TEMPERATURE_UPDATE=n
+
+# TODO: Workaround that disables factory data until it will be supported.
+CONFIG_CHIP_FACTORY_DATA=n
+CONFIG_CHIP_FACTORY_DATA_BUILD=n
+
+# TODO: Workaround that disables CRACEN and switch to OBERON for now. To be removed once SPAKE2+ will be supported.
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
+
+CONFIG_PSA_USE_CRACEN_KEY_MANAGEMENT_DRIVER=n # Disable ECC_SECP_R1_256 [SPAKE | PASE]
+CONFIG_PSA_USE_CRACEN_ASYMMETRIC_DRIVER=n # Disable ECDSA [Certificates]
+CONFIG_PSA_USE_CRACEN_MAC_DRIVER=n # Disable CMAC/HMAC [Secure session]
+CONFIG_PSA_USE_CRACEN_KEY_AGREEMENT_DRIVER=n # disable ECDH_SECP_R1_256 [CASE]
```

Comment:
This PR: https://github.com/nrfconnect/sdk-nrf/pull/14661 allows using this workaround (without the fix these configs are useless here), but it also fixes Cracen's SPAKE2+ implementation, so those configs and the workaround won't be needed anymore. They can be removed.

Reply:
And we should set `CONFIG_PSA_CRYPTO_DRIVER_OBERON=n` here